src/Type/TrueBooleanType.php

```diff
@@ -3,52 +3,26 @@
 namespace PHPStan\Type;
 
 use PHPStan\TrinaryLogic;
-use PHPStan\Type\Traits\NonCallableTypeTrait;
-use PHPStan\Type\Traits\NonIterableTypeTrait;
-use PHPStan\Type\Traits\NonObjectTypeTrait;
-use PHPStan\Type\Traits\NonOffsetAccessibleTypeTrait;
 
-class TrueBooleanType implements BooleanType
+class TrueBooleanType extends BooleanType
```

Comment:
I don't think this is right from the point of LSP - `bool` is "more", `true` is more narrow and accepts less values then `bool`. Might not be important, just something to think about.